pkgs/data/fonts/montserrat/default.nix

```diff
@@ -1,29 +1,28 @@
-# Originally packaged for ArchLinux.
-#
-# https://aur.archlinux.org/packages/ttf-montserrat/
+{ stdenv, lib, fetchFromGitHub }:
 
-{ lib, fetchzip }:
+stdenv.mkDerivation rec {
```

Comment:
fonts normally do not use mkDerivation but just the fetch part and postFetch to move files.

Reply:
Ok ill re-jig it

Reply:
rejigged